backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -103,15 +103,15 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
     *
     * @return the execution handle for the job.
     */
-  def execute(): ExecutionHandle
+  def execute(): Future[ExecutionHandle]
```

Comment:
I know you touched on this in the main comment but I thought one of the points behind the AsyncBackend actors was to work around the Future based contracts in the standard backend? (Which taken out of context sounds really weird). If we're exposing futures anyways, why not get rid of the Async variant?